OMCompiler/SimulationRuntime/fmi/export/openmodelica/fmu2_model_interface.c.inc

```diff
@@ -455,8 +457,89 @@ fmi2Component fmi2Instantiate(fmi2String instanceName, fmi2Type fmuType, fmi2Str
 
   /* intialize modelData */
   fmu2_model_interface_setupDataStruc(comp->fmuData, comp->threadData);
-  useStream[LOG_STDOUT] = 1;
-  useStream[LOG_ASSERT] = 1;
+
+  char* flags_filename = malloc(strlen(fmuResourceLocation) + 10);
+  flags_filename[0] = 0;
+  strcat(flags_filename, fmuResourceLocation);
+  strcat(flags_filename, "/flags");
+
+  char* flags = 0;
+  long length;
+
+  FILE* file = fopen(flags_filename, "rb");
+
+  if (file)
```

Comment:
What's with the lack of curly braces where needed throughout the file?

Reply:
There are curly brackets right in the next line, i don't really know what you mean? Is it just that you prefer 
```
if (condition) {
 }
```
to
```
if (condition) 
{
 }
```

?